dist/puppetsync/tasks/checkout_git_feature_branch_in_each_repo.rb

```diff
@@ -8,8 +8,8 @@ def checkout_modules_to_branch(branch, repo_paths)
     status = 'created'
     Dir.chdir dir
     warn "== #{dir}"
-    `git branch --contains #{branch} &> /dev/null`
-    if $CHILD_STATUS.success?
+    rev = `git name-rev #{branch} &> /dev/null`
+    if !rev.empty?
```

Comment:
@DavidS : I didn't understand this change: are there situations where `git name-rev` is preferable to `git branch --contains` for this kind of check?